src/GeniusYield/Clb/Params.hs

```diff
@@ -0,0 +1,127 @@
+module GeniusYield.Clb.Params where
+
+import Cardano.Api.Shelley qualified as C (shelleyGenesisDefaults)
+import Cardano.Ledger.Alonzo qualified as L (AlonzoEra)
+import Cardano.Ledger.Alonzo.Core qualified as L (CoinPerWord(CoinPerWord))
+import Cardano.Ledger.Alonzo.Language qualified as Alonzo
+import Cardano.Ledger.Alonzo.PParams qualified as Alonzo
+import Cardano.Ledger.Alonzo.Scripts qualified as Alonzo
+import Cardano.Ledger.Babbage qualified as L (BabbageEra)
+import Cardano.Ledger.Babbage.PParams qualified as Babbage
+import Cardano.Ledger.BaseTypes qualified as L
+import Cardano.Ledger.Coin qualified as L
+import Cardano.Ledger.Core qualified as L
+import Cardano.Ledger.Crypto qualified as L (StandardCrypto)
+import Cardano.Ledger.Shelley.Genesis qualified as L (ShelleyGenesis, mkShelleyGlobals)
+import Cardano.Slotting.EpochInfo qualified as S (fixedEpochInfo)
+import Cardano.Slotting.Time qualified as S (SlotLength)
+import Data.Coerce (coerce)
+import Data.Either (fromRight)
+import Data.Functor.Identity (Identity)
+import Data.Map.Strict qualified as Map
+import Data.Maybe (fromJust)
+
+
+-- | Type that unifies protocol parameters across eras.
+data PParams
+  = -- | alonzo era protocol parameters
+    AlonzoParams (L.PParams (L.AlonzoEra L.StandardCrypto))
+  | -- | babbage era protocol parameters
+    BabbageParams (L.PParams (L.BabbageEra L.StandardCrypto))
+
+babbageOnly :: PParams -> L.PParams (L.BabbageEra L.StandardCrypto)
+babbageOnly (BabbageParams v) = v
+babbageOnly _ = error "Sorry, Babbage only"
+
+-- Alonzo
+
+-- | Default Alonzo era parameters
+defaultAlonzoParams :: PParams
+defaultAlonzoParams = AlonzoParams defaultAlonzoParams'
+
+defaultAlonzoParams' :: L.PParams (L.AlonzoEra L.StandardCrypto)
+defaultAlonzoParams' =
```

Comment:
I dislike hardcoding config values in code. Is there a way to get these from a config file?